hax/treemakers/corrections.py

```diff
@@ -188,7 +263,7 @@ def extract_data(self, event):
 
         # FDC
         # Apply the (new) 3D data driven FDC, using NN positions and TPF positions
-        for algo in ['nn', 'tpf']:
+        for algo in ['nn', 'tpf', 'nn_tf']:
             cvals = [result['x_observed_' + algo], result['y_observed_' + algo], z_observed]
             result['r_correction_3d_' + algo] = self.corrections_handler.get_correction_from_map(
                 "fdc_3d", self.run_number, cvals)
```

Comment:
Should we be using the FDC for TFNN defined in [hax.ini](https://github.com/XENON1T/hax/blob/master/hax/hax.ini#L129)?

Reply:
Yes, this need to be corrected!